Don't check tests which depend on DNS - it's disabled in Koji

diff -urNp a/testing/fulltests/default/T070com2sec_simple b/testing/fulltests/default/T070com2sec_simple
--- a/testing/fulltests/default/T070com2sec_simple	2018-07-18 11:52:56.081185545 +0200
+++ b/testing/fulltests/default/T070com2sec_simple	2018-07-18 11:54:18.843968880 +0200
@@ -134,6 +134,10 @@ SAVECHECKAGENT '<"c406a", 255.255.255.25
 SAVECHECKAGENT 'line 30: Error:' # msg from h_strerror so it varies
 SAVECHECKAGENT 'line 31: Error:' # msg from h_strerror so it varies
 
+FINISHED
+
+# don't test the rest, it depends on DNS, which is not available in Koji
+
 CHECKAGENT '<"c408a"'
 if [ "$snmp_last_test_result" -eq 0 ] ; then
   CHECKAGENT 'line 32: Error:'
diff -urNp a/testing/fulltests/default/T071com2sec6_simple b/testing/fulltests/default/T071com2sec6_simple
--- a/testing/fulltests/default/T071com2sec6_simple	2018-07-18 11:52:56.080185548 +0200
+++ b/testing/fulltests/default/T071com2sec6_simple	2018-07-18 11:55:17.779818732 +0200
@@ -132,6 +132,10 @@ SAVECHECKAGENT '<"c606a", ffff:ffff:ffff
 SAVECHECKAGENT 'line 27: Error:'
 SAVECHECKAGENT 'line 28: Error:'
 
+FINISHED
+
+# don't test the rest, it depends on DNS, which is not available in Koji
+
 # 608
 CHECKAGENT '<"c608a"'
 if [ "$snmp_last_test_result" -eq 0 ] ; then
